src/main/java/com/github/krzysiekjodlowski/ox/Board.java

```diff
@@ -0,0 +1,110 @@
+package com.github.krzysiekjodlowski.ox;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Represents square game board.
+ * @author KrzysiekJodlowski
+ */
+class Board {
+    /**
+     * Size of the board game.
+     */
+    private final int boardCapacity;
+    /**
+     * Minimal boardSize.
+     */
+    private final int minBoardSize = 3;
+    /**
+     * Maximal boardSize (temporary value for now).
+     */
+    private final int maxBoardSize = 1000;
+    /**
+     * Core field in this class.
+     */
+    private final Map<FieldNumber, Symbol> moves = new HashMap<>();
+    /**
+     * Size of one side of the board.
+     */
+    private final int boardSize;
+
+    /**
+     *  Constructor initialization.
+     * @param boardSize defines one board game side length
+     * @throws BoardSizeOutOfBoundsException exception
+     * which should be handled by client
+     */
+    Board(final int boardSize) throws BoardSizeOutOfBoundsException {
```

Comment:
Currently your board does nothing more than storing inserted values and using getters to let other classes see it's state. Maybe consider asking your board do to something with it's private data instead of revealing this data  to other objects ("the one who has the data necessary to do something should be the one to do it")

Reply:
As i wrote 4 comments above Im considering placing BoardPrinter in Board so most of the Board data could be "touched" from inside, without exposing it outside. For example as a helper inner class.

Reply:
Or maybe even give up on that printer and incorporate its method into Board as toString?